quickstarts/nextcloud/config.yml

```diff
@@ -0,0 +1,39 @@
+slug: nextcloud
+
+description: |
+  ## Comprehensive monitoring quickstart for Nextcloud
+  With New Relic's Nextcloud monitoring system, keep an eye on your Nextcloud in real-time, correlate transactions moving across the application environment and troubleshoot the issue which you are facing.
+  ### Why monitor Nextcloud?
+  Control, monitor and secure all the data of Nextcloud with Infrastructure monitoring agent and Prometheus open metric integration to view a thorough Nextcloud server dashboard that contains a high-level overview of their app's performance. 
+  - Assist teams in creating architectural maps, tracking down all the issues fast, and observing the Apdex of the app.
+  - Maintain a quick throughput and low latency.
+  - Control over the workflow of [Nextcloud](https://nextcloud.com/) .
+  - Check the health of the dependencies such as database, application, server.
+  - Give specific results to help with the problems.
+  ### What’s included?
+  The New Relic Nextcloud monitoring quickstart offers rapid observability:
+  - Alerts (heap memory usage, scrape errors).
+  - Dashboards (uptime, free space in bytes, total active users, heap memory usage and more).
+  - Enables administrators to keep track of the status and operation of a Nextcloud server installation.
```

Comment:
"Enable your Nextcloud admins to keep track of the status and operation of your server installations."